cmd/plasmacli/subcmd/root.go

```diff
@@ -29,15 +29,15 @@ func RootCmd() *cobra.Command {
 	}
 
 	rootCmd.AddCommand(
-		tx.TxCmd(),
-		eth.EthCmd(),
-		query.QueryCmd(),
+		tx.RootCmd(),
```

Comment:
need to update TxCmd in subcmd/tx/root.go

Reply:
ah woops, fixed